### ISSUE TITLE
Install Che on azure fail.  Che pod restart everytime.

### ISSUE THREAD

**Issue by 9suns** [eclipse-che/che#15460]
install che on azure with below command
``` bash
chectl server:start --installer=helm --platform=k8s --tls --domain=medigger.xyz --multiuser --k8spodwaittimeout=600000 --k8spodreadytimeout=660000 --cheboottimeout=90000 --cheimage=eclipse/che-server:7.3.3 --che-operator-image=quay.io/eclipse/che-operator:7.3.3
```
and with the result:
```
  ✔ Verify Kubernetes API...OK
  ✔ 👀  Looking for an already existing Che instance
    ✔ Verify if Che is deployed into namespace "che"
      ✔ Found running che deployment
      ✔ Found running postgres deployment
      ✔ Found running keycloak deployment
      ✔ Found running plugin registry deployment
      ✔ Found running devfile registry deployment
 ›   Warning: Deployed Che is found and the specified installation parameters will be ignored
  ✔ 👀  Starting already deployed Che
    ✔ Scaling up Che Deployments...done.
  ❯ ✅  Post installation checklist
    ❯ Che pod bootstrap
      ✔ scheduling...done.
      ✔ downloading images...done.
      ✖ starting
        → Error: Client network socket disconnected before secure TLS connection was established
      Retrieving Che Server URL
      Che status check
Error: Error: Client network socket disconnected before secure TLS connection was established
    at KubeHelper.wrapK8sClientError (/usr/local/lib/chectl/lib/api/kube.js:1367:20)
    at KubeHelper.<anonymous> (/usr/local/lib/chectl/lib/api/kube.js:541:28)
    at Generator.throw (<anonymous>)
    at rejected (/usr/local/lib/chectl/node_modules/tslib/tslib.js:108:69)
```
check the pods status
``` shell
$ kubectl get pods --namespace che
# NAME                                READY   STATUS    RESTARTS   AGE
# che-5b6775c7fd-5j4kd                0/1     Running   17         79m
# devfile-registry-6668bcfc46-686qj   1/1     Running   0          79m
# keycloak-8b45dbcd8-5j7nr            1/1     Running   0          79m
# plugin-registry-565cbd5d8b-rvgr5    1/1     Running   0          79m
# postgres-ccf9d6c88-pb47p            1/1     Running   0          79m
```
check the che-5b6775c7fd-5j4kd log around restart
[che_log.xlsx](https://github.com/eclipse/che/files/3951399/che_log.xlsx)

this is my deploy env:
``` shell
$ chectl version
# chectl/7.4.0 linux-x64 node-v10.17.0
$ kubectl version
# Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.0", GitCommit:"70132b0f130acc0bed193d9ba59dd186f0e634cf", GitTreeState:"clean", BuildDate:"2019-12-07T21:20:10Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"linux/amd64"}
# Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.12", GitCommit:"524c3a1238422529d62f8e49506df658fa9c8b8c", GitTreeState:"clean", BuildDate:"2019-11-14T05:26:24Z", GoVersion:"go1.11.13", Compiler:"gc", Platform:"linux/amd64"}
$ helm version
# Client: &version.Version{SemVer:"v2.16.1", GitCommit:"bbdfe5e7803a12bbdf97e94cd847859890cf4050", GitTreeState:"clean"}
# Server: &version.Version{SemVer:"v2.16.1", GitCommit:"bbdfe5e7803a12bbdf97e94cd847859890cf4050", GitTreeState:"clean"}
```

Please help me, thanks

**Comment by benoitf** [comment]
hello, it looks like strange to use `helm` installer together with `--che-operator-image=quay.io/eclipse/che-operator:7.3.3` as it's for the operator installer


for your error, it could be proxy error or TLS support
are you using a proxy ?

**Comment by 9suns** [comment]
I used ```helm``` because of the install guild.
I installed server time, totally failure in the "Che pod bootstrap" step，lots of time tells me timeout for waiting the pod, and few times this “socket disconnected” , may be the hotel network I used has something problem.
And I don't think this caused by the network, before the error occurred, It took about ten minutes because of my timeout setting, can we focus on the restart status and the log, or what can I give to you to solve this problem?
